examples/most-redis-docker/README.md

```diff
@@ -0,0 +1,132 @@
+# Most + Redis + Docker
+
+A suite of simple apps to illustrate how to use most to handle redis events.
+
+This suite may be run using [docker](https://www.docker.com) or [node.js
+10.x](https://nodejs.org/en/).  If you have docker installed already or would
+like to try it, follow the docker instructions.  Otherwise, you will need to be
+running redis locally.  Follow the node instructions to install node and redis.
+
+## Install
+
+### Pre-requisites
+
+1. Install [git](https://help.github.com/articles/set-up-git/)
+2. Clone this repo:
+
+```sh
+git clone git@github.com:mostjs/core.git
+cd examples/most-redis-docker
+```
+
+### Assumptions
+
+The remainder of this document assumes your working directory is
+`most-redis-docker`.  You may have to modify some of the following directions
+if you chose another folder name.
+
+This project uses environment variables.  If you are using the windows command
+prompt, you may have to change some commands slightly to use env vars. For
+example, rather than type
+`REDIS_MYCHANNEL=most-redis-docker-example npm run consume`, you would type
+`set REDIS_MYCHANNEL=most-redis-docker-example&&npm run consume`.
+
+### Option 1: Docker
+
+1.  Install [docker](https://www.docker.com/get-started).  (You will need to
+    create a [DockerHub](https://hub.docker.com) account.)
+2.  Build the docker image for this project:
+
+```sh
+docker build -t most-redis-docker .
```

Comment:
I changed the name of the docker image (here and in docker-compose.yml) to better reflect the name of this example.

Reply:
:+1: